Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -1473,9 +1475,9 @@ - (void)testSetClientResetMode {
     RLMRealmConfiguration *config = [user configurationWithPartitionValue:partitionValue clientResetMode:RLMClientResetModeDiscardLocal];
```

Comment:
I left the deprecated discardLocal in some obj-c and swift tests so that I could be sure I wasn't breaking it, just deprecating. I imagine once it's removed from the api we can completely scrub it from all tests.